spec/components/find_interface/courses/entry_requirements_component/view_preview.rb

```diff
@@ -17,10 +17,22 @@ def fully_populated
       render FindInterface::Courses::EntryRequirementsComponent::View.new(course: mock_course)
     end
 
+    def fully_etp_populated
```

Comment:
Can we call it `fully_populated_with_etp_course` or something?

Reply:
done